systemtests/src/main/java/io/enmasse/systemtest/messaginginfra/ResourceManager.java

```diff
@@ -139,96 +130,66 @@ public MqttClientFactory getMqttClientFactory() {
     // Common resource methods
     //------------------------------------------------------------------------------------------------
 
-    public <T extends HasMetadata> void createResource(T... resources) {
+    private final ResourceType<?>[] resourceTypes = new ResourceType[]{
+            new MessagingInfraResourceType(),
+            new MessagingTenantResourceType(),
+    };
+
+    @SafeVarargs
+    public final <T extends HasMetadata> void createResource(T... resources) {
+        createResource(true, resources);
+    }
+
+    @SafeVarargs
+    public final <T extends HasMetadata> void createResource(boolean waitReady, T... resources) {
```

Comment:
@lulf maybe add overloaded method with waithReady as default? but it is can be added in future

Reply:
ah nevermind it is there